i2c/simulation/i2c_sim.cpp

```diff
@@ -14,12 +14,22 @@ auto SimI2C::central_transmit(uint8_t *data, uint16_t size,
     std::copy_n(data, size, last_transmitted.begin());
     transmit_count++;
     if (!sensor_map.empty()) {
-        uint16_t store_in_register = 0;
-        auto *iter = data + 1;
-        iter = bit_utils::bytes_to_int(iter, data + size, store_in_register);
-        LOG("Storing %d in register %d", store_in_register, reg);
-        sensor_map[dev_address].REGISTER_MAP[reg] = store_in_register;
         next_register_map[dev_address] = reg;
+        if (size > 1) {
+            auto *iter = data + 1;
+            uint16_t store_in_register = 0;
+            // check if address belongs to pressure sensor
+            if (dev_address == 0x67 << 1) {
```

Comment:
The i2c simulator could use some upgrades so you don't have to do stuff like this. We should have simulated i2c devices and route data based on device address. The register map is not sufficient.

I'm not asking you to change anything.